src/gui/PluginBrowser.cpp

```diff
@@ -205,7 +205,9 @@ PluginDescWidget::PluginDescWidget(const PluginKey &_pk,
 	setFixedHeight( DEFAULT_HEIGHT );
 	setMouseTracking( true );
 	setCursor( Qt::PointingHandCursor );
-	setToolTip(_pk.description());
+	setToolTip(_pk.desc->subPluginFeatures
+		? _pk.description()
+		: tr(_pk.desc->description));
```

Comment:
Although this will certainly not break anything, I am not sure whether this will be effective since `_pk.desc->description` is dynamic.

Reply:
In the case that `_pk.desc->subPluginFeatures` is null, `_pk.desc` is a static descriptor from a plugin binary, e.g.: https://github.com/LMMS/lmms/blob/f34f894a84f9885b91da295f8ee676fb96b3c1e1/plugins/Amplifier/Amplifier.cpp#L34-L45

I'm not aware of any code that modifies these (we should probably make them const), and the description is clearly intended to be translated. This also matches the code in `EffectSelectDialog`: https://github.com/LMMS/lmms/blob/f34f894a84f9885b91da295f8ee676fb96b3c1e1/src/gui/EffectSelectDialog.cpp#L237